Tasks/MavenV3/maventask.ts

```diff
@@ -565,4 +568,20 @@ function execBuildWithRestore() {
         execBuild();
     }
 }
+
+function replaceGifToBase64() {
```

Comment:
Could you please add comments to new method (jsdoc)?

Reply:
Add comment to all of 3 methods.